packages/ui-gui/package.json

```diff
@@ -1,28 +1,35 @@
 {
   "name": "@webpack-cli/ui-gui",
   "version": "0.0.1",
+  "license": "MIT",
   "description": "GUI for webpack-cli ui",
-  "main": "index.js",
+  "main": "src/index.js",
   "scripts": {
-    "build": "webpack --config=webpack.prod.js",
-    "serve": "webpack-dev-server --config=webpack.prod.js"
+    "start": "if-env NODE_ENV=production && npm run -s serve || npm run -s dev",
+    "build": "preact build",
+    "serve": "preact build && preact serve",
+    "dev": "preact watch",
+    "lint": "eslint src"
   },
-  "license": "MIT",
+  "eslintConfig": {
+    "extends": "eslint-config-synacor",
+    "rules": {
+      "no-unused-vars": "warn",
+      "react/sort-comp": "off",
+      "lines-around-comment": "off",
+      "react/prefer-stateless-function": "off"
+    }
+  },
+  "eslintIgnore": [
+    "build/*"
+  ],
   "devDependencies": {
-    "@babel/core": "^7.1.2",
-    "@webpack-cli/init": "^0.1.2",
-    "babel-core": "^6.26.3",
-    "babel-loader": "^8.0.4",
-    "babel-plugin-syntax-dynamic-import": "^6.18.0",
-    "babel-preset-env": "^1.7.0",
-    "css-loader": "^1.0.0",
-    "mini-css-extract-plugin": "^0.4.4",
-    "node-sass": "^4.9.3",
-    "sass-loader": "^7.1.0",
-    "style-loader": "^0.23.1",
-    "webpack": "^4.21.0",
-    "webpack-cli": "^3.1.2",
-    "webpack-dev-server": "^3.1.10"
+    "if-env": "^1.0.0",
```

Comment:
Crossenv here instead if this package is responsible for env

Reply:
Also the `README.md` of `if-env`, they recommend to install `per-env`. Mmmmh 😅

Reply:
@evenstensberg @ematipico replaced `if-env` for `cross-env`.

Reply:
@evenstensberg @ematipico thinking more about, `cross-env` is not what we expect using here, since `if-env` do a `if/else` to run different command based in which env was defined, that's not exactly what `cross-env` is proposed to do, I'll change that for `per-env` as suggested in `if-env` repo.